frc42_dispatch/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "frc42_dispatch"
 description = "Filecoin FRC-0042 calling convention/dispatch support library"
-version = "5.0.0"
+version = "5.1.0"
```

Comment:
These should be major versions (minor FVM versions -> Major versions everywhere else due to breaking changes).